MYabrv.bib

```diff
@@ -95,6 +96,7 @@ @String{NODe
 @String{OOPSLA = "Proc.\ Conf.\ on Object-Oriented Programming, Systems, Languages and Applications (OOPSLA)"}
 @String{PCS = "Proc.\ Int'l Conf.\ on Economic, Technical and Organisational Aspects of Product Configuration Systems"}
 @String{PLDI = "Proc.\ ACM SIGPLAN Conf.\ on Programming Language Design and Implementation (PLDI)"}
+@String{PMLR = "Proc.\ Int'l Conf.\ on Machine Learning (PMLR)"}
```

Comment:
Is that the correct acronym? I know that conference as ICML?

Reply:
Yes, you are right. I found the paper on this website: https://proceedings.mlr.press/v70/raghu17a.html
and was a bit confused. I changed the conference abbreviation to ICML but kept PLMR as publisher. I am not entirely sure if this is correct though.

Reply:
The fix is now in commit [6bfe622](https://github.com/SoftVarE-Group/BibTags/pull/70/commits/6bfe622a232667ad353ba5154b5ef51a68a14c2d). (I accidentally performed the fixup on the wrong commit.)